infra/app/storekeys.bicep

```diff
@@ -0,0 +1,63 @@
+param keyVaultName string = ''
+param storageAccountName string = ''
+param azureOpenAIName string = ''
+param azureCognitiveSearchName string = ''
+param rgName string = ''
+param formRecognizerName string = ''
+param contentSafetyName string = ''
+param storageAccountKeyName string = 'AZURE-STORAGE-ACCOUNT-KEY'
+param openAIKeyName string = 'AZURE-OPEN-AI-KEY'
+param searchKeyName string = 'AZURE-SEARCH-KEY'
+param formRecognizerKeyName string = 'AZURE-FORM-RECOGNIZER-KEY'
+param contentSafety string = 'AZURE-CONTENT-SAFETY-KEY'
+
+
+resource storageAccountKeySecret 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: keyVault
+  name: storageAccountKeyName
+  properties: {
+    value: listKeys(resourceId(subscription().subscriptionId, rgName, 'Microsoft.Storage/storageAccounts', storageAccountName), '2021-09-01').keys[0].value
+  }
+}
+
+resource openAIKeySecret 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: keyVault
+  name: openAIKeyName
+  properties: {
+    value: listKeys(resourceId(subscription().subscriptionId, rgName, 'Microsoft.CognitiveServices/accounts', azureOpenAIName), '2023-05-01').key1
+  }
+}
+
+resource searchKeySecret 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: keyVault
+  name: searchKeyName
+  properties: {
+    value: listAdminKeys(resourceId(subscription().subscriptionId, rgName, 'Microsoft.Search/searchServices', azureCognitiveSearchName), '2021-04-01-preview').primaryKey
+  }
+}
+
+resource formRecognizerKeySecret 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: keyVault
+  name: formRecognizerKeyName
+  properties: {
+    value: listKeys(resourceId(subscription().subscriptionId, rgName, 'Microsoft.CognitiveServices/accounts', formRecognizerName), '2023-05-01').key1
+  }
+}
+
+resource contentSafetyKeySecret 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  parent: keyVault
+  name: contentSafety
+  properties: {
+    value: listKeys(resourceId(subscription().subscriptionId, rgName, 'Microsoft.CognitiveServices/accounts', contentSafetyName), '2023-05-01').key1
+  }
+}
+
+resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' existing = {
+  name: keyVaultName
+}
+
+output CONTENT_SAFETY_KEY string = contentSafetyKeySecret.name
+output FORM_RECOGNIZER_KEY string = formRecognizerKeySecret.name
+output SEARCH_KEY string = searchKeySecret.name
+output OPENAI_KEY string = openAIKeySecret.name
+output STORAGE_ACCOUNT_KEY string = storageAccountKeySecret.name
```

Comment:
You cannot export keys from a bicep module.  You'll have to get them when they are needed in the module they are needed in.

Reply:
The keys here are the key names stored in the `KeyVault`, not the actual key values.